test/common/router/config_impl_test.cc

```diff
@@ -3555,6 +3555,45 @@ name: foo
   }
 }
 
+TEST(RouteConfigurationV2, DirectResponse) {
```

Comment:
For new tests please add a `//` comment with a brief description of what the test does.